cli/src/command/account_manager.rs

```diff
@@ -46,11 +45,11 @@ pub enum AccountManagerCommand {
 
 #[derive(Debug, Clone, Args)]
 pub struct InitParameters {
```

Comment:
does this need an `#[command(author, version)` as well?

Reply:
No